### PR TITLE
Add key points and objectives

### PR DESCRIPTION
**Pull request by ErinBecker**
As per: https://github.com/datacarpentry/wrangling-genomics/issues/25

### CODE REVIEW

File: _episodes/02-variant_calling.md [hunk 1]
```diff
@@ -3,13 +3,15 @@ title: "Variant Calling Workflow"
...
+- "How do I find sequence variants between my samples and a reference genome?"
 objectives:
-- "Make use of command line tools to perform a variant calling workflow."
 - "Describe the steps involved in variant calling."
 - "Describe the types of data formats encountered during variant calling."
+- "Use command line tools to perform variant calling."
 keypoints:
-- "First key point."
+- "Bioinformatics command line tools are collections of commands that can be used to carry out bioinformatics analyses."
+- "To use most powerful bioinforatics tools, you'll need to use the command line."
```
Comment (crazyhottommy) [inline _episodes/02-variant_calling.md]: typo here: bioinformatics

**Comment by ErinBecker** [comment]
Thanks @crazyhottommy. I've fixed the typo. Please go ahead and merge if it looks good to you.

**Comment by crazyhottommy** [comment]
@ErinBecker good to me. merged

**Comment by ErinBecker** [comment]
Thanks @crazyhottommy!

**Pull request merged**